frontend/src/old-pages/Configure/Source.js

```diff
@@ -181,7 +194,13 @@ function Source() {
               {
                 value: "wizard",
                   label: "Wizard",
-                  description: "Choose this to start a new cluster configuration."
+                  description: <Box margin={{bottom: "xs"}} >
```

Comment:
![Screenshot 2022-07-11 at 11 39 18](https://user-images.githubusercontent.com/3091286/178236187-a821a5eb-e36e-465e-8784-5c93747c451d.png)
issue: I personally find very confusing this kind of UX where I can pick a template from the list but I also have another option called in the same way (Template) beneath it. While this is not a major issue it still needs some UX attention, and can be reworked in different ways:
- rename the Template section: maybe with a less fancy name like "Wizard with existing configuration"?
- rename the new templates feature: presets?
- add another option for our users: Empty wizard / Wizard with preset / Wizard from existing configuration / and so on

@sean-smith what do you think?

Reply:
I think having two Wizard sections might be confusing but I could re-work the template section to be either "Start from an existing template" or "upload your own template" maybe with a toggle UI element, or a setting in the list "Upload a default template". Definitely want some PM eyes on this feature.